src/tx/applyTx/checkSpendCond.js

```diff
@@ -27,6 +27,7 @@ const {
   ERC20_BYTECODE,
   ERC721_BYTECODE,
   ERC1948_BYTECODE,
+  ERC1948_BYTECODE_218508104,
```

Comment:
I thought that different bytecodes are effective since different heights.. E.g. `ERC1948_BYTECODE_41000` is effective till height 41000, `ERC1948_BYTECODE_55000` (imaginary next switch) is effective from 41000 to 55000 and so on 🤔

Reply:
the condition first checks chainID, then checks height, i didn't want to put both in there.